src/main/java/com/WebController/FrontController.java

```diff
@@ -0,0 +1,48 @@
+package com.WebController;
+
+import com.Model.Patient;
+import com.Repository.EventRepository;
+import com.Repository.PatientRepository;
+import com.Model.Event;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Controller
+public class FrontController {
+
+    @Autowired
+    private PatientRepository patientRepo;
+    @Autowired
+    private EventRepository eventRepo;
+
+    @GetMapping("/")
+    public String homePage() {
+        return "index";
+    }
+
+    @PostMapping(value = "/patientinfo", consumes = "application/json")
+    @ResponseBody
+    public String receivePatient (@RequestBody List<Event> events){
```

Comment:
this is probably good enough for now, but I wonder if in the future we want to be able to add events one by one to the same patient. I realize patients are anonymous, but within a session we should be able to know which patient id we are talking about,